py3-cairo.yaml

```diff
@@ -80,13 +80,26 @@ subpackages:
             python: python${{range.key}}
             import: ${{vars.import}}
         - runs: |
-            include="$(python${{range.key}} -c 'import ${{vars.import}}; print(${{vars.import}}.get_include())')"
+            include="$(python${{range.key}} -c 'import cairo; print(cairo.get_include())')"
             test -f "$include/py3cairo.h"
       environment:
         contents:
           packages:
             - py3-supported-${{vars.pypi-package}}
 
+  - range: py-versions
+    name: py${{range.key}}-${{vars.pypi-package}}-dev
+    description: py${{range.key}} dev files for ${{vars.pypi-package}}
+    dependencies:
+      runtime:
+        - cairo-dev
+        - py${{range.key}}-${{vars.pypi-package}}
+      provides:
+        - py3-${{vars.pypi-package}}-dev
+      provider-priority: ${{range.value}}
+    pipeline:
+      - uses: split/dev
```

Comment:
When I build this locally, these are empty.

Reply:
yeah, i guess. and i guess i'm fine to drop this then. 
i'll drop it.
thank you for paying attention.